config/environments/development.rb

```diff
@@ -1,8 +1,6 @@
 # frozen_string_literal: true
 
 Rails.application.configure do
-  # Verifies that versions and hashed value of the package contents in the project's package.json
-  config.webpacker.check_yarn_integrity = true
```

Comment:
We don't need this anymore? or is this flag set somewhere else?

Reply:
That doesn't appear in the rails 6 environment files anymore, and I believe it's because the preferred approach is to add the integrity check to webpacker.yml instead, and we have it there for the development environment.

Reply:
👍

Reply:
oh that's interesting.